debian/changelog

```diff
@@ -0,0 +1,6 @@
+imx-firmware (5.10.72-2.2.2-1) buster; urgency=medium
+
+  [ Ramiro Gsponer ]
```

Comment:
You don't need `[ Ramiro Gsponer ]` if this is the only entry in the changelog and it was made by you